mw-webapp/src/globalStore/ServiceWorkerStore.ts

```diff
@@ -121,7 +122,7 @@ class ServiceWorker {
    * Update current permissions in instance
    */
   private getNotificationPermissions = () => {
-    const isNotificationAvailable = "Notification" in window;
+    const isNotificationAvailable = localStorageWorker.getItemByKey<SettingPageSettings>("settingPage")?.isNotificationAllowed;
```

Comment:
Should we use some specific place to check notification avalilability?  Or at least check and save to localstorage on each app loading? For now it looks a little dangerous

Reply:
Done